core/modules/modPrune.class.php

```diff
@@ -223,10 +223,9 @@ public function init($options = '')
 	 * @param   string  $options    Options when enabling module ('', 'noboxes')
 	 * @return  int                 1 if OK, 0 if KO
 	 */
-	public function remove($options = '')
-	{
+	public function remove($options= '') {
 		$sql = [];
 
-		return $this->_remove($sql, $options);
+		return $this->_remove($sql,$options);
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma
**Severity**: ERROR
**File**: core/modules/modPrune.class.php L229

No space found after comma in argument list

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2008274925>*